pi-manage

```diff
@@ -586,6 +586,25 @@ def create(name, scope, action):
     return r
 
 
+@policy_manager.command
+def create_from_file(name, filename):
```

Comment:
It will only break backward compatibility to your unofficial patch. But it will be consisten in the parameter naming.